### ISSUE TITLE
Request: Filter to only show on self

### ISSUE THREAD

**Issue by kaedys** [AdiAddons/AdiButtonAuras#185]
Many types of buffs, it would be useful to allow it to only highlight if **you** have the buff, rather than your target.  Flasks are a good example, it's not useful information to highlight the flask on my bar of my target has the buff.  The only concern I have is whether I have it (there are other raid management addons out there for tracking who is missing flasks, and in any case, it only shows that type of flask, not *any* flask).  Lone Wolf buffs are similar, they show green if *anyone in the group* has that raid buff, rather than if I have it.  Given that it's an aura, it isn't terribly useful information to tell me if *at least one raid member* has a similar buff.

Basically, it'd be nice for most buff-type effects if I could select an option to only check me, not the target or the whole raid.

**Comment by Rainrider** [comment]
Lone Wolf (and raid buffs in general):
If you see a number there it means there is at least one person in your group missing a buff from that category. This is an indication you should recast. Keep in mind that actively cast abilities take precedence, so maybe someone else should recast. This behavior won't change.

Items buffs:
I could fix that to track flasks only on self. Are there any other items that need special handling? All consumables?

**Comment by exGeoScottL** [comment]
Consumables would be my preference, as well as buffs that are self-only (like Chakras, though I've not tested if they check target or not.  They may already only check the player).  Note that they probably shouldn't check classes of consumables still.  For example, if I have a flask of agility active, it shouldn't light up both the flask of agility and the flask of stamina (or whatever).

As for Lone Wolf, the issue is that noticing a small green number in a green-outlined box is a lot harder than noticing a lack of a green box entirely.  I have an 8-button bar setup in the upper right next to my buff frame that has each of the Lone Wolf buffs in it.  Previously, with InlineAuras, I knew which ones I was lacking, because they weren't lit up.  Now, it'll light all of them up if anyone in the group has that buff, even if I don't, which makes it kinda a pain to figure out which aura to pop if I, for example, get rezed mid-fight and get incomplete buffs.  All I'm asking for is an optional checkbox to allow it to only check self.

On a side note, I believe it pops up the number even if everyone has the buff, if they only have an aura buff and there's a cast one available.  For example, if I pop the Mastery LW aura, but there's a paladin in the group that can cast Might, the number will show up even though everyone really does have Mastery currently (the aura just being less reliable than a durationed buff).  I'm not contesting this behavior, just pointing out that the presence of a number in the button doesn't necessarily mean I (or someone else) doesn't currently have that raid buff.  It could also mean they have an aura version and could have a castable one instead.

Really, though, the request is to give me some method of being able to glance at my Lone Wolf bar and see which raid buffs I'm missing, and thus an at-a-glance indicator of which aura I should pop for my own personal damage if I get rezed with incomplete buffs.

**Comment by Rainrider** [comment]
I pushed an alpha version to wowace. Helpful cosumable items should now only check the player. This may have implications on Drums but I'll check into that when I have more time.

Flasks still only look for their own buff and I find this usefull, so I won't combine them.

I'll take a look at Lone Wolf when time permits but I still think it is working as intended.

**Comment by Rainrider** [comment]
Raid buffs are gone now.
